blueprint/index.md

```diff
@@ -0,0 +1,169 @@
+category: 6
+summary: |
+  This Genesys Cloud Developer Blueprint demonstrates how to integrate the Genesys Cloud Javascript Platform SDK into your React app.  This example uses Typescript, although the implementation will be similar if you're using plain javascript.  The sample app provides examples of use cases such as creating a profile page for a user and admin operations like signing all the users out of a queue.  The blueprint will describe the steps needed to integrate the Genesys Cloud SDK into your React app.
+---
+
+<!-- no toc -->
+* [Sample Application Features](#sample-application-features "Goes to the Sample Application Features section")
+* [Solution Components](#solution-components "Goes to the Solutions Components section")
+* [Requirements](#requirements "Goes to the Requirements section")
+* [Running Locally](#running-locally "Goes to the Running Locally section")
+* [Sample App Overview](#sample-app-overview "Overview of the sample app's features")
+* [Configuring the React Project to use Genesys Cloud SDK](#configuring-the-react-project-to-use-genesys-cloud-sdk "How to integrate the Genesys Cloud SDK")
+* [Additional resources](#additional-resources "Goes to the Additional resources section")
+
+
+![React App Flowchart](images/flowchart.png)
+
+## Sample Application Features
+
+- **User Home Page** - Displays meaningful information about the present user
+- **User Search** - Allows a user to dynamically search for other Genesys Cloud users by name or email.
+- **User Queues** - Displays information for the queues the user belongs to and provides an admin option to log out all users from the queue.
+
+## Solution Components
+
+- **React Application** - The React application that will be setup to use the Genesys Cloud Javascript Platform SDK
+- **TypeScript** - Used alongside the React application in the sample app
+
+### Software Development Kit (SDK)
+
+- **Genesys Cloud Javascript Platform SDK** - Providing the steps for integrating this SDK and using it in meaningful ways is the focus of the sample app.
+
+## Requirements
+
+### Specialized knowledge
+
+Implementing this solution requires experience in several areas or a willingness to learn:
+
+- Genesys Cloud Platform API knowledge
+- React knowledge
+- Typescript knowledge
+
+### Genesys Cloud account requirements
+
+This solution requires a Genesys Cloud license. For more information on licensing, see [Genesys Cloud Pricing](https://www.genesys.com/pricing "Opens the pricing article").
+
+A recommended Genesys Cloud role for the solutions engineer is Master Admin. For more information on Genesys Cloud roles and permissions, see the [Roles and permissions overview](https://help.mypurecloud.com/?p=24360 "Opens the Roles and permissions overview article").
+
+## Running Locally
+
+### Download the repository containing the project files
+Go to the [repository](https://github.com/GenesysCloudBlueprints/react-app-with-genesys-cloud-sdk) and clone it to your machine.
```

Comment:
use the repo name in the link text so users aren't guessing what they're going to go to